skypy/halos/tests/test_sampler.py

```diff
@@ -0,0 +1,32 @@
+import numpy as np
+import colossus as colossus
+from colossus.lss import mass_function
```

Comment:
```suggestion
```
Again we will import this later to avoid any errors if colossus is not installed